src/Audio_details.md

```diff
@@ -0,0 +1,192 @@
+
+Then, both of these two get their respective volume scaled, once from [`NR50`](<#FF24 — NR50: Master volume & VIN panning>), and once from the volume knob (if the console has one).
+Note that the former step never mutes a non-silent input, but the latter can.
+
+Each of the two analog outputs then goes through a [high-pass filter](https://en.wikipedia.org/wiki/High-pass_filter) (HPF).
+For short, a HPF constantly tries to "pull" the signal towards analog 0 (neutral); the reason for that is explained further below.
+
+[^digital_analog]:
+To be clear: digital values are discrete and clear-cut; conversely, the analog domain is continuous.
+The former is what computers use, the latter is what the real world is made of.
+
+[^vin]:
+Actually, VIN acts as a 5<sup>th</sup> channel fed into the mixer, whose control bits are in NR50 instead of NR51.
+This was omitted from the diagram for simplicity.
+
+## PCM registers
+
+These two registers, only present in the Game Boy Color and ulterior models, allow reading the output of the generation circuits directly; this is very useful for testing "internal" APU behavior.
+These registers are not documented in any known Nintendo manual.
+
+### FF76 — PCM12 (CGB Mode only): Digital outputs 1 & 2 \[read-only\]
+
+The low nibble is a copy of sound channel 1's digital output, the high nibble a copy of sound channel 2's.
+
+### FF77 — PCM34 (CGB Mode only): Digital outputs 3 & 4 \[read-only\]
+
+Same, but with channels 3 and 4.
+
+## Finer technical explanation
+
+### DIV-APU
+
+A "DIV-APU" counter is increased every time `DIV`'s bit 4 (5 in [double-speed mode](<#FF4D — KEY1 (CGB Mode only): Prepare speed switch>)) goes from 1 to 0, therefore at a frequency of 512 Hz (regardless of whether double-speed is active).
+Thus, the counter can be made to increase faster by writing to `DIV` while its relevant bit is set (which clears `DIV`, and triggers the falling edge).
+
+Some events are tied to the DIV-APU counter:
+
+Event          | Every N ticks | Frequency[^div_apu_freq]
+---------------|---------------|-------------------------
+Envelope sweep | 8             | 64 Hz
+Sound length   | 2             | 256 Hz
+CH1 freq sweep | 4             | 128 Hz
+
+[^div_apu_freq]:
+Indicated values are under normal operation; the frequencies will obviously differ if writing to `DIV` to increase the counter faster.
+
+### Mixer
+
+A high-pass filter (HPF) removes constant biases over time.
+The HPFs therefore remove the DC offset created by inactive channels with an enabled DAC, and off-center waveforms.
+
+Enabling or disabling a DAC ([see below](#DACs)), adding or removing it using NR51, or changing the volume in NR50, will cause an audio pop.
+(All of these actions cause a change in DC offset, which is smoothed out by the HPFs over time, but still creates a pop.)
+
+The HPF is more aggressive on GBA than on GBC, which itself is more aggressive than on DMG.
```

Comment:
This should be reworded in a way that doesn't require the explanation of what it means to be aggressive *afterwards*.

Reply:
That would break the flow of the comparison and make it hard to read. Introducing the concept first would be confusing, as in "why am I being explained this?".